docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[Kubernetes-dashboard]]
+= The Kubernetes dashboard: Linux session data from your Kubernetes clusters
+
+This page explains how to deploy a https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/[daemonset] to collect session data from your Kubernetes clusters using the Endpoint and Cloud Security integration, and how to view that data in Kibana’s Kubernetes dashboard.
+
+**Prerequisites**: This feature requires Elastic Stack version 8.4 or higher. Your Elastic deployment must have the Endpoint and Cloud Security integration enabled, and you need an active Fleet Server.
+
+**Support Matrix**: This feature is currently available on GKE and EKS using Linux hosts and Kubernetes versions that match the following specifications:
+|=====================
+| | **Kubernetes versions** | **Container runtime** | **Nose OSes**
+|**EKS**| 1.18; 1.19; 1.20; 1.21| Docker | Amazon Linux 2, Bottlerocket OS
+|**GKE**| Regular (default channel): 1.21 and 1.22; Stable: 1.20 and 1.21; Rapid: 1.22 and 1.23 | containerd | Container-optimized OS (COS), Ubuntu
+|=====================
+You first need to download the daemonset manifest .yaml, then modify it to include your Fleet URL and Enrollment Token before you deploy it to the clusters you want to monitor. After that, your data will appear in the Kubernetes dashboard.
+
+[discrete]
+== Download and modify the daemonset manifest
+The daemonset integrates Elastic Endpoint into your Kubernetes cluster. The Elastic Agent is enrolled to a running Fleet Server using the `FLEET_URL` parameter, and connected to a specific Elastic Agent policy using the `FLEET_ENROLLMENT_TOKEN`.
```

Comment:
I like that we are saying `specific Elastic Agent policy`; however, it may not be clear that the policy we are targeting _**must have**_ the Endpoint integration added to it. Does it make sense to add some language to that effect for precision and to reduce confusion?

Reply:
I also agree with Norrie on this, I also would like to note that for this release, we want to turn the beta flag for `session_data=true`, that can only be done within the Endpoint policy settings page

Reply:
💯

Reply:
:100:, incorporating these changes.